frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -374,7 +378,10 @@ const TermPlanner = () => {
                 <UnplannedColumn
                   dragging={!!draggingCourse}
                   courseInfos={Object.fromEntries(
-                    planner.unplanned.map((code) => [code, courseInfos[LIVE_YEAR][code]])
+                    planner.unplanned.map((code) => [
+                      code,
+                      courseInfos[Math.min(validYears.at(-1)!, LIVE_YEAR)][code]
```

Comment:
```suggestion
                      courseInfos[validYears.includes(LIVE_YEAR) ? LIVE_YEAR : validYears.at(-1)!][
                        code
                      ]
```

or 

```suggestion
                      LIVE_YEAR in courseInfos
                        ? courseInfos[LIVE_YEAR][code]
                        : courseInfos[validYears.at(-1)!][code]
```
otherwise, if we start degree after LIVE_YEAR we get index error :c